interface/patient_file/pos_checkout_normal.php

```diff
@@ -307,38 +383,45 @@ function imdeleted() {
                             );
                         }
                         ?>
-                        <tr>
-                            <td colspan='5'>&nbsp;</td>
-                        </tr>
-                        <tr>
-                            <td><?php echo text(oeFormatShortDate($svcdispdate ?? '')); ?></td>
-                            <td><b><?php echo xlt('Total Charges'); ?></b></td>
-                            <td class='text-right'>&nbsp;</td>
-                            <td class='text-right'>&nbsp;</td>
-                            <td class='text-right'><?php echo text(oeFormatMoney($charges, true)) ?></td>
+                        <!--<tr>
+                            <td style="border:none !important" colspan='5'>&nbsp;</td>
+                        </tr>-->
```

Comment:
hi @omegasystemsgroup , should remove this commented code

Reply:
Alright